r: patch both .conda and .tar.bz2 pkgs

awwad opened the pull request:
This fixes the problem of ignoring .conda packages, which was
occurring for some of the patching being done (namespace, etc.).

repodata contains both 'packages' and 'packages.conda' dicts,
and both must be attended to.

dbast commented:
closing and reopening to trigger newly added github action to see the diff.

dbast commented:
@awwad Did not rebase to solve the conflicts to not break the signature of the signed commits. Instead merged master and fixed the lint findings on top of it. Feel free to rebase and overwrite.

awwad commented:
Updated to exclude the patch-instructions.json part of the fix, which seemed to break something in the hotfixing process on Zeus specifically.  There's a follow-up ticket for it.  Repodata hotfixing is still fixed, though.

Rebased, picked off lint, and re-signed.  Testing on Zeus.  Saw something slightly odd, so holding off again.

varlackc reviewed (commented):
The following `PR` has been open for a few years, maybe we should either merge it or close it at this point.

ryanskeith commented:
Closing due to staleness
